### PR TITLE
Add known issues for collapsed navbar 

### PR DESCRIPTION
**Pull request by Monkeychip**
Document the known issue for when the navbar is in a collapsed state and users cannot click on the dev console or namespace picker.

### CODE REVIEW

File: website/content/partials/known-issues/ui-collapsed-navbar.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Collapsed navbar does not allow you to click inside the console or namespace picker
+
+#### Affected versions
+
+This issue affects Vault versions 1.14.0+ and 1.15.0+. A fix for this version will be released in Vault 1.16.0.
```
Comment (schavis) [inline website/content/partials/known-issues/ui-collapsed-navbar.mdx:5]:
```suggestion
The UI issue affects Vault versions 1.14.0+ and 1.15.0+.
A fix is expected for Vault 1.16.0.
```

Style correction: avoid "this" as a pronoun

File: website/content/partials/known-issues/ui-collapsed-navbar.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+#### Issue
+
+The version of HDS the UI is using does not allow users to click inside the dev console or namespace picker when accessing these components from the collapsed view, triggered in smaller viewports.
```
Comment (schavis) [inline website/content/partials/known-issues/ui-collapsed-navbar.mdx:9]:
```suggestion
The Vauil UI currently uses a version of HDS that does not allow users to click
within collapsed elements. In particular, the dev console or namespace picker
become inaccessible when viewing the components in smaller viewports.
```

Style correction: 1 idea per sentence, write in active voice

File: website/content/partials/known-issues/ui-collapsed-navbar.mdx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+#### Workaround
+
+Expand the width of the screen until the full navbar is displayed, and click the components from within this state.
```
Comment (schavis) [inline website/content/partials/known-issues/ui-collapsed-navbar.mdx:13]:
```suggestion
Expand the width of the screen until you deactivate the collapsed view. Once the full navbar is displayed, click the desired components.
```

**Review by schavis — CHANGES_REQUESTED** [review]

**Comment by Monkeychip** [comment]
@schavis Thank you for the suggestions! It's really nice to have someone helping with the wording and tone.

**Review by schavis — APPROVED** [review]

**Pull request merged**